commands/previewPush.go

```diff
@@ -129,13 +153,30 @@ func (args *PushArgs) flags() []cli.Flag {
 	return flags
 }
 
+const pwarn = "WARNING: This command will receive a major upgrade in the next release (v4.15). Try the new behavior by using ppreview/ppush. Please test and report any bugs ASAP. The older code will be available as oldpreview/oldpush for a limited number of releases."
+const owarn = "WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead."
```

Comment:
Shouldn't this be `oldpush`?

```suggestion
const owarn = "WARNING: oldpreview/oldpush are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead."
```


```diff
$ dnscontrol oldpush
-[WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead.]
+[WARNING: oldpreview/oldpush are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead.]
```